diff --git a/node_modules/vite/dist/node/chunks/dep-79892de8.js b/node_modules/vite/dist/node/chunks/dep-79892de8.js
index 5fc674d..3288e7c 100644
--- a/node_modules/vite/dist/node/chunks/dep-79892de8.js
+++ b/node_modules/vite/dist/node/chunks/dep-79892de8.js
@@ -54014,7 +54014,7 @@ async function instantiateModule(url, server, context = { global }, urlStack = [
 // In node@12+ we can use dynamic import to load CJS and ESM
 async function nodeImport(id, importer, resolveOptions) {
     let url;
-    if (id.startsWith('node:') || isBuiltin(id)) {
+    if (id.startsWith('node:') || id.startsWith('data:') || isBuiltin(id)) {
         url = id;
     }
     else {
