diff --git a/node_modules/@solid-primitives/map/dist/index.js b/node_modules/@solid-primitives/map/dist/index.js
index bee801a..dc5d886 100644
--- a/node_modules/@solid-primitives/map/dist/index.js
+++ b/node_modules/@solid-primitives/map/dist/index.js
@@ -1,4 +1,4 @@
-import { batch } from 'solid-js';
+import { batch } from 'solid-js/dist/solid.js';
 import { TriggerCache } from '@solid-primitives/trigger';
 
 // src/index.ts
